downloads.ini

```diff
@@ -94,3 +94,12 @@ sha256 = b93b73572c5e495154a9823d494de5729c77d1c83b041171154c4b5f3f76b590
 extractor = 7z
 output_path = third_party/node/win
 strip_leading_dirs=node-v%(version)s-win-x64
+
+[clang]
+version = n353803-99ac9ce7-1
+# Use this only if clang bundled with LLVM is outdated
+# List of all versions https://commondatastorage.googleapis.com/chromium-browser-clang/index.html?path=Win/
+url = https://commondatastorage.googleapis.com/chromium-browser-clang/Win/clang-%(version)s.tgz
+download_filename = clang-%(version)s.tgz
+output_path = third_party/llvm-build/Release+Asserts
```

Comment:
This is confusing me. Why doesn't this `clang` download conflict with the `llvm` download, since they have the same output path?

Reply:
The clang in the llvm package is outdated.  This was added to the download to override the lib and bin folder from the llvm package with the clang version needed for the build to work.  When I was building the package before this I had to do this manually.

Reply:
clang overwrites buggy clang from llvm. I didn't look into downloads.py to see what happens if files already exist on extraction location, but I gave it a shot and it worked.

Reply:
I understand that you need to override Clang provided by `llvm`. However, I am concerned by how you implemented it because you have two downloads with the exact same `output_path`. How can I be certain that files from `clang` will override `llvm`?

Also if you're going to download `clang` from Google, why bother mixing differing versions of Clang and the rest of the LLVM toolchain? Wouldn't it be better to just commit fully to Google's toolchain until LLVM builds catch up?

Reply:
I just went though a few more of the gni files and it seems like the build script is only getting the default_clang_base_path from that folder and does not use the rest of the llvm package we were downloading.  If we comment out the llvm section from the downloads.ini file and just use the new clang from google the build should still work. I have it building now and it is past the part it used to crash at before the switch to the google clang.

Reply:
Works for me. FYI the only reason I'd want to go back to using the llvm.org builds eventually is there's a slight philosophical problem using Google's binaries in ungoogled-chromium...

Reply:
I think I can solve the google binary problem.  It looks like llvm just released version 11.0.0-rc1 of the windows binaries. 
I removed the [clang] section and changed llvm version number to 11.0.0-rc1 and uncommented out the stable version url to github and it seems to work.  I am currently building but it is pass where is used to crash and seeems to work.
[llvm]
version = 11.0.0-rc1
#Uncomment when pre-release version is used.
#url = https://prereleases.llvm.org/win-snapshots/LLVM-%(version)s-win64.exe
#Uncomment the below instead when a new enough stable version of LLVM comes around
url = https://github.com/llvm/llvm-project/releases/download/llvmorg-%(version)s/LLVM-%(version)s-win64.exe
download_filename = LLVM-%(version)s-win64.exe
extractor = 7z
output_path = third_party/llvm-build/Release+Asserts

Reply:
I updated llvm in the latest commit and left google's clang commented to be there if necessary in future.
I didn't test it myself tho.